### PR TITLE
Update API's to support group call

### PR DESCRIPTION
**Pull request by priyankaaprakash**
## Purpose

- Add group call API
- Add BYOI API's
- Update Azure dependency
- Add sample buttons to the app to use the new API's

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Run the sample app based on the new README.md

## What to Check
Verify that the following are valid

- user can join a meeting
- user can join a group call

### CODE REVIEW

File: TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java [hunk 1]
```diff
@@ -18,37 +19,46 @@
...
 import java.util.concurrent.Callable;
 
-public class MainActivity extends AppCompatActivity implements MeetingUIClientEventListener, MeetingUIClientIdentityProvider {
+public class MainActivity extends AppCompatActivity implements MeetingUIClientEventListener,
+        MeetingUIClientIdentityProvider, MeetingUIClientUserEventListener {
 
     private final String ACS_TOKEN = "<ACS_TOKEN>";
     private final String meetingUrl = "<MEETING_URL>";
     private final String displayName = "John Smith";
+    private final UUID groupId = UUID.fromString("29228d3e-040e-4656-a70e-890ab4e173e5");
```
Comment (rasinive) [inline TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java:42]: Remove the UUID and replace with <GROUP_ID>

File: TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java [hunk 2]
```diff
@@ -120,15 +141,46 @@ public void onRemoteParticipantCountChanged(int newCount) {
...
+    @Override
+    public void provideSubTitleFor(String userIdentifier, MeetingUIClientIdentityProviderCallback meetingUIClientIdentityProviderCallback) {
+        if (isACSUser(userIdentifier)) {
+            meetingUIClientIdentityProviderCallback.onSubTitleAvailable("ACS Subtitle");
+        }
+    }
+
+    @Override
+    public void onNamePlateOptionsClicked(Activity activity, String userIdentifier) {
+        Toast.makeText(getApplicationContext(), "Name plate options clicked:", Toast.LENGTH_SHORT).show();
```
Comment (rasinive) [inline TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java:172]: nit: remote :

File: TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java (hunk 1 above)
Comment (priyankaaprakash) [reply TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java]: Fixed

File: TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java (hunk 2 above)
Comment (priyankaaprakash) [reply TeamsEmbedAndroidGettingStarted/app/src/main/java/com/microsoft/TeamsEmbedAndroidGettingStarted/MainActivity.java]: Fixed

**Review by rasinive — APPROVED** [review]

**Pull request merged**